Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -1212,6 +1212,31 @@ static constexpr Array<@type@, @size@> @name@ { {
     append_code_point_display_names("BlockNameData"sv, "s_block_display_names"sv, unicode_data.block_display_names);
     append_code_point_display_names("CodePointName"sv, "s_code_point_display_names"sv, unicode_data.code_point_display_names);
 
+    {
+        constexpr size_t max_bidi_types_per_row = 20;
```

Comment:
Is there some kind of rule about how many things to put per line? I copied this from elsewhere. We don't remotely fit 20x "BidirectionalCharacterType" on a line, but I'm not seeing the other generated data fit onto lines either.

Reply:
Oh it's *very* scientific. I pick a per_row number that isn't so high as to make my text editor stop working (Sublime Text seems to disable syntax highlighting / clangd after some outrageous line length 🤷‍♂️ ). And a number that isn't so low as to make the generated file a billion lines long.

So, real answer: it doesn't matter, I just try to make my local tools not hate me too much.